api/core/workflow/nodes/trigger_webhook/node.py

```diff
@@ -112,13 +120,42 @@ def _get_normalized(mapping: dict[str, Any], key: str) -> Any:
 
             if param_type == "file":
                 # Get File object (already processed by webhook controller)
-                file_obj = webhook_data.get("files", {}).get(param_name)
-                outputs[param_name] = file_obj
+                files = webhook_data.get("files", {})
+                if files and isinstance(files, dict):
+                    file = files["file"]
+                    if file and isinstance(file, dict):
+                        try:
+                            related_id = file.get("related_id")
+                            transfer_method_value = file.get("transfer_method")
+                            if not transfer_method_value:
+                                raise ValueError("transfer_method is required in file mapping")
```

Comment:
I suggest use dedicated exception for this errors. `ValueError` is too ambiguous and making refactoring harder.

Reply:
change it to key error